### DISCUSSION TITLE
SA1 games

### DISCUSSION THREAD

**Discussion by iceyman** [sanni/cartreader#350] [Need help]
Just wondering what is the best way to read / dump SA1 games in version 4 thanks

**Comment by sanni** [comment]
You need the Adafruit Clock Generator and the flashed snesCIC installed.
Then you turn on the Cart Reader and try to dump the SNES game and if you get an header error you switch it off and on and try dumping again. Repeat until you get past the header error, it might take up to 10 tries.

SA1 had this weird behavior since the beginning, maybe in the future someone will attach a logic analyzer to the Cart Reader and find out why it is so random. Once it is completely unlocked it will always read perfectly(until you switch it off again). So I suspect the CIC part inside the SA1 chip to be the culprit rather than the part that does the actual mapping.

**Reply by iceyman** [reply]
THanks for tha fast reply
